src/modules/delete.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-mutable-exports */
+export let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+export default function deleteTask() {
+  const deleteBtn = document.querySelectorAll('.option');
+  deleteBtn.forEach((btn) => {
+    const container = document.querySelector('.container');
+    btn.addEventListener('click', (e) => {
+      const item = e.target.parentNode.parentNode;
+      container.removeChild(item);
+      const deletedTask = e.target.previousElementSibling.textContent;
+      tasks = tasks.filter((task) => task.description !== deletedTask);
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    });
+  });
+}
```

Comment:
- [ ] removing a task gives an error in the console and it removes the entire local storage as you can see in below print screen.
![Screenshot 2022-04-29 203426](https://user-images.githubusercontent.com/39240820/166004822-15746ac8-9261-449c-b77e-6c526c305686.png)
- [ ] keep in mind that Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

Reply:
when I open the app locally I'm not getting that `error` but on the deployed one is where I'm getting it. I don't really know what I can do to solve that. could you mind helping @mikethreels? 

The app is working fine I think, the user can add, and remove. tasks are saved in the local storage.

Reply:
The `error` of the local storage to be deleted I just fix it now. 

> About that error that is thrown in the console I've been looking all over the internet I can't find a way out. And you may notice in the dynamically rendered element, the element I'm removing is actually a child od the container buy I don't understand how I'm getting that error...

Reply:
Hi Christian,
Sorry for the late response, at the time you wrote to me I wasn't able to respond, great job fixing it 👏